charts/incubator/deemix/questions.yaml

```diff
@@ -0,0 +1,339 @@
+    group: "Controller"
+    label: ""
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+        - variable: advanced
+          label: "Show Advanced Controller Settings"
+          schema:
+            type: boolean
+            default: false
+            show_subquestions_if: true
+            subquestions:
+              - variable: type
+                description: "Please specify type of workload to deploy"
+                label: "(Advanced) Controller Type"
+                schema:
+                  type: string
+                  default: "deployment"
+                  required: true
+                  enum:
+                    - value: "deployment"
+                      description: "Deployment"
+                    - value: "statefulset"
+                      description: "Statefulset"
+                    - value: "daemonset"
+                      description: "Daemonset"
+              - variable: replicas
+                description: "Number of desired pod replicas"
+                label: "Desired Replicas"
+                schema:
+                  type: int
+                  default: 1
+                  required: true
+              - variable: strategy
+                description: "Please specify type of workload to deploy"
+                label: "(Advanced) Update Strategy"
+                schema:
+                  type: string
+                  default: "Recreate"
+                  required: true
+                  enum:
+                    - value: "Recreate"
+                      description: "Recreate: Kill existing pods before creating new ones"
+                    - value: "RollingUpdate"
+                      description: "RollingUpdate: Create new pods and then kill old ones"
+                    - value: "OnDelete"
+                      description: "(Legacy) OnDelete: ignore .spec.template changes"
+# Include{controllerExpert}
+  - variable: env
+    group: "Container Configuration"
+    label: "Image Environment"
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+        # Include{fixedEnv}
+        - variable: PUID
+          label: "PUID"
+          description: "User ID of the user you want the container to run as in order to fix folder permission issues"
+          schema:
+            type: int
+            default: 1000
+        - variable: PGID
+          label: "PGID"
+          description: "Group ID, see PUID"
+          schema:
+            type: int
+            default: 1000
+        - variable: UMASK_SET
+          label: "UMASK_SET"
+          description: "Setting UMASK for file permissions Default is 022"
+          schema:
+            type: string
+            default: "022"
+        - variable: INTPORT
+          label: "Internal Port"
+          description: "This changes the internal port of deemix. DON'T CHANGE THIS IF YOU DONT KNOW WHY YOU WOULD DO IT"
+          schema:
+            type: int
+          default: 3333
```

Comment:
This should be moved to values.yaml ONLY.
And be set to 
INTPORT: "{{ .Values.service.main.ports.main.targetPort }}"

If you leave it in questions it will set the apps port to 3333 and break the service.
Instead dynamic assign the port based on targetPort, as I showed above :)